Spanner/tests/System/PgWriteTest.php

```diff
@@ -1107,4 +1116,18 @@ public function testExecuteUpdateBatchMultipleErrors()
         $this->assertEquals(Code::INVALID_ARGUMENT, $res->error()['status']['code']);
         $this->assertEquals($statements[2], $res->error()['statement']);
     }
+
+    private function assertValues($expected, $actual, $delta = 0.000001)
+    {
+        if (is_float($expected)) {
+            $this->assertEqualsWithDelta($expected, $actual, $delta);
```

Comment:
Why is this delta? Could you please point me to a documentation for this?

Reply:
FLOAT values should always be compared with a delta, as the decimal representation is an approximate representation of the binary value. So there will always be rounding when we're comparing decimal representations, which is handled by the delta. FLOAT64 values did not need a delta as most decimal numbers are representable in the 64bit namespace, but 32bit floating points will see more rounding (e.g. 1.1 is not representable in 32bit -- it is stored as 1.10000002 internally)